docs/projects.md

```diff
@@ -9,19 +9,19 @@ Projects allow you to track dependencies for a given software repository (e.g. a
 To create a new project and associate the working directory with it, simply run: 
```

Comment:
I'd say the filename and title of this documentation is still accurate, but I'd be curious if you've considered changing it? Since it's talking about projects in general, I would say that "project" is weird.

Reply:
I did consider it, but decided not to. As you point out, this page is talking about the general topic of "Projects" and not the specific subcommand `project`.

The work that @furi0us333 is doing will restructure our docs around the subcommands and at that point it will rightfully be `project` (singular).